massa-hash/src/error.rs

```diff
@@ -4,7 +4,7 @@ use displaydoc::Display;
 use thiserror::Error;
 
 #[non_exhaustive]
-#[derive(Display, Error, Debug)]
+#[derive(Display, Error, Debug, Clone)]
```

Comment:
Can we (in a followup issue)  make Error enums all derive the same traits?

Reply:
Follow up